src/av2/evaluation/tracking/utils.py

```diff
@@ -9,10 +9,11 @@
 from typing import Any, Dict, Iterable, List, Optional, Union, cast
 
 import numpy as np
+import numpy.typing as npt
 from tqdm import tqdm
 
-from av2.utils.typing import NDArrayInt
-
+NDArrayFloat = npt.NDArray[np.float64]
```

Comment:
Like in the previous file, these types have already been defined in a common file. Please use the existing definition. :-)